### ISSUE TITLE
[BUG] unable to edit network configuration during installation 

### ISSUE THREAD

**Issue by CyborTronik** [harvester/harvester#3446]
**Describe the bug**

Unable to jump between editing fields during fresh installation on bare metal. 

**To Reproduce**
1. Fresh installation of harvester 1.1.1
2. During installation was able to select the disc to install to
3. Set the hostname
4. Then at the 'Configure Network' section was able to select the Management NIC
5. Not able to jump on other fields in the 'Configure Network' and if I press enter (leaving remaining fields with their defaults - which is supposed to work) - I get instantly returned to the first step about 'Choose instalation mode' 

**Expected behavior**

I would suspect that it should go to the next step or start the installation 

**Support bundle**

**Environment**
 - Harvester ISO version: 1.1.1
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630): Bare metal Dell T7810 

**Additional context**

**Comment by ibrokethecloud** [comment]
this is a known issue: https://github.com/harvester/harvester/issues/2708

are you please able to try one of the work arounds or try PXE based installs?

**Comment by xeruf** [comment]
Nope, this issue is different: In the other one, the popup to select the NIC does not properly open.
We are able to select the NIC, but not proceed to the other steps.

As for the workaround: I am in a Java console (jviewer.jnlp), can open the NIC selection, but have no way to confirm my selection and move on:

![image](https://github.com/harvester/harvester/assets/13354331/8456ec25-c057-4f1b-a2e1-f415ceb5fc27)

**Comment by xeruf** [comment]
Tried with 1.1.2, issue persists

**Comment by xeruf** [comment]
Somehow it worked now by pressing enter, let's see if this is reliable now...

**Comment by tserong** [comment]
@xeruf I know this is an old issue now - I wanted to check if you had any further feedback. If not, I'll close this in favour of #2708.

**Comment by rrajendran17** [comment]
@xeruf  can we close this ?
